iothub_client/tests/iothubtransport_amqp_cbs_auth_ut/iothubtransport_amqp_cbs_auth_ut.c

```diff
@@ -1430,23 +1439,6 @@ TEST_FUNCTION(authentication_do_work_first_auth_times_out)
     authentication_destroy(handle);
 }
 
-// Tests_SRSIOTHUBTRANSPORT_AMQP_AUTH_09_097: [If `authentication_handle` or `name` or `value` is NULL, authentication_set_option shall fail and return a non-zero value]
```

Comment:
>Tests_SRSIOTHUBTRANSPORT_AMQP_AUTH_09_097 [](start = 3, length = 41)

Make sure all requirements removed from code are also removed from tests and requirement docs. AMQP transport has separate .md files for each module.